roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -141,7 +143,9 @@
     dest: "{{ tempdir }}/templates/oauth-client.yml"
   vars:
     kibana_hostname: "{{ openshift_logging_kibana_hostname }}"
+    kibana_ops_hostname: "{{ openshift_logging_kibana_ops_hostname }}"
```

Comment:
Looks like that is no longer true... it had worked initially, i wonder if the module changed its functionality